### ISSUE TITLE
Reviews don't show when using SSH based Gerrit remotes 

### ISSUE THREAD

**Issue by talios** [uwolfer/gerrit-intellij-plugin#12]
_Updated ticket after diagnosing the problem_.

When a git repository is configured to talk to Gerrit via SSH, the IntelliJ Plugin fails to resolve the project names to search for. My `.git/config` file contains:

```
[remote "gerrit"]
  url = gerrit:dev/smx3.schema
  push = develop
```

Which is resolved via SSH's `~/.ssh/config` file:

```
Host gerrit
  HostName gerrit.smxemail.com
  Port 29418
  User markd
```

Even when the `.git/config` file uses the FQN domain name, review projects are not detected.

**Comment by cg-d** [comment]
Feedback to @talios 
I had the same problem when using a ssh remote which is resolved via ssh config. A simple workaround is to add a 2nd remote which uses the FQN of the gerrit server.

> git remote -v
gerrit	ssh://cgd@gerrit.myserver.de:29418/Project (fetch)
gerrit	ssh://cgd@gerrit.myserver.de:29418/Project (push)
origin	ssh://gerrit_d/Project (fetch)
origin	ssh://gerrit_d/Project (push)

**Comment by lilianmoraru** [comment]
@cg-d Adding a second remote didn't work for me...
What I ended up doing is cloning the repository in a temporary location and then manually applying the patch(`git pull`)...
I definitely cannot change the settings in `.ssh/config`, the entire monorepo relies heavily on that alias.

Update: Adding a second remote Did work, I had to force the IDE to reload the project.

**Comment by uwolfer** [comment]
This should be fixed with PR #324. Please reopen if it does not fix your situation after the next release of this plugin.
